cucumber-core/src/main/java/io/cucumber/core/order/factory/LexicalUriPickleOrderFactory.java

```diff
@@ -0,0 +1,17 @@
+package io.cucumber.core.order.factory;
+
+import io.cucumber.core.order.PickleOrder;
+import io.cucumber.core.order.StandardPickleOrders;
+
+public class LexicalUriPickleOrderFactory implements PickleOrderFactory {
```

Comment:
Please make all classes `final`. This will allow us to make changes in the future without breaking peoples code unexpectedly.